pkg/psmdb/statefulset.go

```diff
@@ -13,6 +13,8 @@ import (
 
 	api "github.com/percona/percona-server-mongodb-operator/pkg/apis/psmdb/v1"
 	"github.com/percona/percona-server-mongodb-operator/pkg/naming"
+	"github.com/percona/percona-server-mongodb-operator/pkg/psmdb/pmm"
+	"github.com/percona/percona-server-mongodb-operator/pkg/psmdb/psmdbconfig"
```

Comment:
The PR looks good. The only thing I don't like is the name of this package.  In my opinion, it would be more idiomatic to avoid repeating names and use `.../pkg/psmdb/config` instead of `.../pkg/psmdb/psmdbconfig`